### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by LeManhCuong35**
Init Rails project with mysql2

### CODE REVIEW

**Comment by LeManhCuong35** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.2'
+
+gem 'rails', '~> 6.1.7', '>= 6.1.7.3'
```
Comment (ghost) [inline Gemfile]: Dùng "

**Review by ghost — COMMENTED** [review]
cmt

**Comment by LeManhCuong35** [comment]
ready

**Comment by LeManhCuong35** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**